src/components/ui/LoginForm.tsx

```diff
@@ -28,12 +30,29 @@ export const LoginForm = () => {
   }, [checkSession]);
 
   if (isLoading) {
-    return <div>Loading...</div>;
+    return (
+      <button
+        disabled
+        className="bg-white border border-gray rounded flex items-center justify-center h-10 px-8"
+        onClick={() => signInWithRedirect()}
+      >
+        <div className="flex items-center justify-center h-5 mr-2 w-5">
+          <Icon icon={GoogleSigninIcon} color="white" size={4} />
+        </div>
+        <span> Login in with Google</span>
+      </button>
+    );
   }
 
   return (
-    <div className="login-form">
-      <button onClick={() => signInWithRedirect()}>Login</button>
-    </div>
+    <button
+      className="bg-white border border-gray rounded flex items-center justify-center h-10 px-8"
+      onClick={() => signInWithRedirect()}
+    >
+      <div className="flex items-center justify-center h-5 mr-2 w-5">
+        <Icon icon={GoogleSigninIcon} color="white" size={4} />
+      </div>
+      <span> Login in with Google</span>
+    </button>
```

Comment:
I suppose button has a bit of room for improvement!

- Better to use common `Button` component instead of pure html button tag
- The `div` tag to wrap `Icon` seems unnecessary?
- `Login` text's font size, line-height and text itself are different from the ones in the design.

⬇️ This is how I would implement, please refer it if needed!
```tsx
   <Button
      size="md"
      className="bg-white border border-gray gap-4 px-14"
      onClick={() => signInWithRedirect()}
    >
      <Icon icon={GoogleSigninIcon} color="white" size={5} />
      <span className="text-lg leading-6">Log in with Google</span>
    </Button>
```

Please share with me if you have any concern 🙏

Reply:
I replaced this with other component (`<Authenticator socialProviders={['google']} hideSignUp={true} />;`).
Thank you for your review...!!